apps/cli/src/commands/shell.ts

```diff
@@ -31,7 +32,7 @@ export default class Shell extends Command {
         const ext2 = path.join(containerDir, path.basename(ext2Path));
         const ramSize = "128Mi";
         const driveLabel = "root";
-        const sdkImage = "sunodo/sdk:0.2.0"; // XXX: how to resolve sdk version?
+        const sdkImage = "sunodo/sdk:0.4.0"; // XXX: how to resolve sdk version?
```

Comment:
Do we need a backport/fix for `0.11.x`?

Running `sunodo shell` with `0.11.x` version will use `sunodo/sdk:0.2.0`.

Reply:
I don't think so